ui/yarn.lock

```diff
@@ -4,20 +4,20 @@
 
 "@adobe/css-tools@^4.0.1":
   version "4.2.0"
-  resolved "https://registry.yarnpkg.com/@adobe/css-tools/-/css-tools-4.2.0.tgz#e1a84fca468f4b337816fcb7f0964beb620ba855"
```

Comment:
This is nitpicky but would you mind reverting this file and re-updating with `yarn install` ?  It's hard to see the actual change here due to all the path updates.  I'm guessing you ran `npm install`, right?  Both of these tools seem to overwrite the other's paths, unfortunately.

I don't know enough to have an opinion about whether `npm install` is better or worse than `yarn install` so I'm open to persuasion here - as long as we have consistency.

Reply:
I think it's a good callout and not nitpicky at all!

I used `npm install` followed by `yarn install` to fix my mistake but I should have reverted the file first.

I updated the PR and force-pushed since the changes were getting messy. The lock file changes are significantly more readable now.

Reply:
Thanks!  This PR looks reasonable overall.